tools/shared.py

```diff
@@ -494,6 +494,9 @@ def array_contains_any_of(hay, needles):
     c_opts += ['-D__SSE4_2__=1']
 
   if array_contains_any_of(user_args, SIMD_INTEL_FEATURE_TOWER[6:]):
+    c_opts += ['-D__SSE4_2__=1']
```

Comment:
The check `if array_contains_any_of(user_args, SIMD_INTEL_FEATURE_TOWER[5:]):` already implies the check `array_contains_any_of(user_args, SIMD_INTEL_FEATURE_TOWER[6:]):`, so this check is redundant.

Reply:
ok